api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -380,6 +380,9 @@ message JwtProvider {
 message JwtCacheConfig {
   // The unit is number of JWTs, default to 100.
   uint32 jwt_cache_size = 1;
+
+  // Max token size in bytes, default to 4096.
```

Comment:
Can you please write the units (I'm assuming "bytes").
Also it is unclear to me if this is the max token size or max cache size (for all tokens). If it is the latter, I suggest renaming the field's name. If the former, this seems like a non-typical cache use, so I suggest adding comments how this cache is working, and making sure the field's comment reflects the correct behavior.
Please also add whether 0 is a valid value, and if so, what is the expected behavior in this case (or consider adding a minimum value constraint for the field).

Reply:
> Can you please write the units (I'm assuming "bytes").

"Max token size in bytes" - isn't it clear that the unit is byte?``

> Also it is unclear to me if this is the max token size

Ok, I changed the comment, I hope it's clear now that it applies to a single token.